dist/index.js

```diff
@@ -1 +1 @@
ern must be a valid regex: "+e.message)}}}},310:e=>{function process(e,t){return e.flatMap(e=>{const r=e.versions.filter(e=>t.versionPattern.test(e.version));const s=r.slice(t.keep);return s.map(t=>({name:e.name,...t}))})}e.exports={process:process}},869:(e,t,r)=>{const{getOctokit:s}=r(438);async function getPackages(e){const t=`\n    query getVersions($owner: String!, $repo: String!, $names: [String!]!) {\n      repository(owner: $owner, name: $repo) {\n        name\n        packages(first: 20, names: $names) {\n          nodes {\n            name\n            versions(last: 100, orderBy: {field: CREATED_AT, direction: DESC}) {\n              nodes {\n                id\n                version\n              }\n            }\n          }\n        }\n      }\n    }\n  `;const r=`\n  query getVersion($owner: String!, $repo: String!, $names: [String!]!, $version: String!) {\n    repository(owner: $owner, name: $repo) {\n      name\n      packages(first: 20, names: $names) {\n        nodes {\n          name\n          version(version: $version) {\n            id\n            version\n          }\n        }\n      }\n    }\n  }\n  `;const o=e.version?r:t;const n=await s(e.token).graphql(o,{owner:e.owner,repo:e.repo,names:e.names,headers:{Accept:"application/vnd.github.package-deletes-preview+json"}});let i;if(e.version){i=n.repository.packages.nodes.map(e=>({...e,versions:[e.version]}))}else{i=n.repository.packages.nodes.map(e=>({...e,versions:e.versions.nodes}))}return i}async function deletePackage(e,t){const r=`\n    mutation deletePackageVersion($packageVersionId: String!) {\n      deletePackageVersion(input: {packageVersionId: $packageVersionId}) {\n        success\n      }\n    }\n  `;await s(t.token).graphql(r,{packageVersionId:e,headers:{Accept:"application/vnd.github.package-deletes-preview+json"}})}e.exports={getPackages:getPackages,deletePackage:deletePackage}},877:module=>{module.exports=eval("require")("encoding")},357:e=>{"use strict";e.exports=require("assert")},614:e=>{"use strict";e.exports=require("events")},747:e=>{"use strict";e.exports=require("fs")},605:e=>{"use strict";e.exports=require("http")},211:e=>{"use strict";e.exports=require("https")},631:e=>{"use strict";e.exports=require("net")},87:e=>{"use strict";e.exports=require("os")},622:e=>{"use strict";e.exports=require("path")},413:e=>{"use strict";e.exports=require("stream")},16:e=>{"use strict";e.exports=require("tls")},835:e=>{"use strict";e.exports=require("url")},669:e=>{"use strict";e.exports=require("util")},761:e=>{"use strict";e.exports=require("zlib")}};var __webpack_module_cache__={};function __webpack_require__(e){if(__webpack_module_cache__[e]){return __webpack_module_cache__[e].exports}var t=__webpack_module_cache__[e]={exports:{}};var r=true;try{__webpack_modules__[e].call(t.exports,t,t.exports,__webpack_require__);r=false}finally{if(r)delete __webpack_module_cache__[e]}return t.exports}__webpack_require__.ab=__dirname+"/";return __webpack_require__(351)})();
```

Comment:
Let's keep this file out of the PR to make the diff smaller. I'll rebuild it on release.